src/docfx/restore/Restore.cs

```diff
@@ -91,7 +91,18 @@ private static void RestorePackages(string docsetPath, Config config, string? lo
             using var packageResolver = new PackageResolver(docsetPath, config, fetchOptions);
             ParallelUtility.ForEach(
                 GetPackages(config, locale, repository).Distinct(),
-                item => packageResolver.DownloadPackage(item.package, item.flags),
+                item =>
```

Comment:
Is it possible to put these code inside `PackageResolver` instead of handling it here? This way the interface stays the same as before. I'd like to keep the entry code cleaner so it's easier to see the structure. `EnsureLocalizationContributionBranch` can possibly be extracted to a `LocUtility` though to share.

Reply:
sounds good to me